lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -316,6 +316,8 @@ export class ResponseHandler {
             responseScopes = ScopeSet.fromString(cacheRecord.accessToken.target).asArray();
             expiresOn = new Date(Number(cacheRecord.accessToken.expiresOn) * 1000);
             extExpiresOn = new Date(Number(cacheRecord.accessToken.extendedExpiresOn) * 1000);
+        } else {
+            throw ClientAuthError.createAccessTokenEntityNullError();
```

Comment:
This might cause a problem for angular with the b2c issue where it returns a blank access token: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/src/msal.interceptor.ts#L62

It might be better to wait until b2c fixes that issue before we include this specific change, given there could be apps (and wrapper libraries) that are relying upon a blank access token to be returned, instead of a new error.

Reply:
Do we know when that fix is going to get implemented? In the meantime that means we DO have scenarioes where no access token is returned.

Reply:
This quarter, I believe. And yes, it means we do, although its technically a bug and not by design.